jetty-core/jetty-http2/jetty-http2-client/src/main/config/modules/http2-client.mod

```diff
@@ -0,0 +1,20 @@
+[description]
+Adds the Jetty HTTP/2 client dependencies to the server classpath.
+
+[tags]
+client
+http2
+
+[files]
+maven://org.eclipse.jetty/jetty-alpn-client/${jetty.version}/jar|lib/jetty-alpn-client-${jetty.version}.jar
+maven://org.eclipse.jetty/jetty-alpn-java-client/${jetty.version}/jar|lib/jetty-alpn-java-client-${jetty.version}.jar
+maven://org.eclipse.jetty.http2/jetty-http2-client/${jetty.version}/jar|lib/http2/jetty-http2-client-${jetty.version}.jar
+maven://org.eclipse.jetty.http2/jetty-http2-common/${jetty.version}/jar|lib/http2/jetty-http2-common-${jetty.version}.jar
+maven://org.eclipse.jetty.http2/jetty-http2-hpack/${jetty.version}/jar|lib/http2/jetty-http2-hpack-${jetty.version}.jar
```

Comment:
The `http2-hpack` and `http2-common` jars are already in jetty-home, so these are not needed.

Reply:
Done.